content/news/core-monthly/2023/core-monthly-2022-12-01-2022-12-31.md

```diff
@@ -0,0 +1,377 @@
+---
+layout: post
+title:  "PrestaShop Core Monthly - December"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2023-01-03
+authors: [ PrestaShop ]
+image: /assets/images/theme/banner-core-monthly.jpg
+twitter_image: /assets/images/theme/banner-core-monthly.jpg
+icon: icon-calendar
+tags:
+ - core-monthly
+---
+
+Release of the autoupgrade module, what's next for PrestaShop 8, the release of the 1.7.8.8, an overview of ongoing work, and much more. This Core Monthly summarizes what happened in the project during December.
+
+![Core Monthly banner](/assets/images/theme/banner-core-monthly-wide.jpg)
+
+Dear developers,
+
+Welcome to the first Core Monthly of 2023! 2022 was a fantastic year for the PrestaShop Project in which you could see [a new major release](/news/2022/prestashop-8-0-0-available/), [announcement of the upcoming new theme](/news/2022/new-theme-announce/) for PrestaShop, and significant changes to the project's structure which made it more open than it ever was.
+
+In 2022 the project welcomed many new members from the community to help it grow. It was also the following year when you could see other project members and me in the [Public Demo](/news/2022/upcoming-demo-9-2022/) and for the first time, discuss with us during the [office hours](/news/2022/office-hours/).
```

Comment:
```suggestion
In 2022, the project welcomed many new members in the community, to help with its growth. It was also the second year where you could see other project members and me in the [Public Demo](/news/2022/upcoming-demo-9-2022/). And for the first time, discuss with us during [office hours](/news/2022/office-hours/).
```

Reply:
@coffrep here, I wanted to write that project welcomed its members to the committers team, not many new members in the community :)

Reply:
Hi @kpodemski , I've just made an update